.github/workflows/add-needs-triage-label.yml

```diff
@@ -0,0 +1,18 @@
+name: Add 'needs-triage' label to new issues
+
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+      
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: andymckay/labeler@1825c391d68ade591efa053af2472e5b0e2e2d9e
```

Comment:
I think `@1.0.3` would also work and would be clearer?

Reply:
Added comment